src/service/topics.ts

```diff
@@ -1,4 +1,5 @@
 import crashlytics from '@react-native-firebase/crashlytics';
+import moment from 'moment';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `moment` library is a heavy library and can increase the bundle size of your application. If you're only using it for simple date manipulations like adding hours, consider using lighter alternatives such as `date-fns` or native JavaScript `Date` object.
```diff
- import moment from 'moment';
+ import { addHours, formatISO } from 'date-fns';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->